test/gtest/ucp/test_ucp_proto_mock.cc

```diff
@@ -159,8 +124,18 @@ class test_ucp_proto_mock : public ucp_test, public mock_iface {
     {
         /* Reset topo provider to force reload from config */
         ucs_sys_topo_reset_provider();
+
+        /*
+         * By default TOPO_PRIO="sysfs,default"
+         * We keep only default config to always have the same topo distances
+         * when test is being executed on different machines.
+         */
         modify_config("TOPO_PRIO", "default");
+
+        /* This test is for dynamic protocol selection available only in v2 */
```

Comment:
maybe we skill the test altogether if protov2 is disabled? since the default is that it's enabled

Reply:
good idea, done